### PR TITLE
Update Cargo.lock

### PR DESCRIPTION
**Pull request by vorot93**
Ran into Cargo.lock conflicts while working on #11540, figured that instead of untangling it, might just update everything.

### CODE REVIEW

**Comment by niklasad1** [comment]
You executed `cargo update`? 

> This has been generally frowned upon previously. Personally I actually don't mind it (better take the hit of random bugs up front than defer to the day it becomes inevitable

I agree. 
It's just hard to review it properly i.e, some crates might have even changed their `MAJOR`  
label (e.g. nom) or bring in other dependencies. However, really nice that it compiles without any `code` changes :tada:

**Comment by ordian** [comment]
test failure on macOS is legit, seems like the test
```
---- v1::tests::mocked::signing_unsafe::rpc_eth_send_transaction_with_bad_to stdout ----
thread 'v1::tests::mocked::signing_unsafe::rpc_eth_send_transaction_with_bad_to' panicked at 'assertion failed: `(left == right)` (left: `Some("{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32602,\"message\":\"Invalid params: 0x prefix is missing.\"},\"id\":1}")`, right: `Some("{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32602,\"message\":\"Invalid params: prefix is missing.\"},\"id\":1}")`, diff: `Some("{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32602,\"message\":\"Invalid params: 0x prefix is missing.\"},\"id\":1}") `)', rpc/src/v1/tests/mocked/signing_unsafe.rs:215:5
```
needs to be updated

I don't mind the update in general, what I don't like though, is that it mixes insubstantial updates with changes like librocksdb-sys 6.2.4 -> 6.6.4, which can have a huge impact.

**Comment by niklasad1** [comment]
> test failure on macOS is legit, seems like the test

So, we are only running the tests on macOS?

@vorot93 

This should fix it:

```git
diff --git a/rpc/src/v1/tests/mocked/signing_unsafe.rs b/rpc/src/v1/tests/mocked/signing_unsafe.rs
index 0c5a81d28..e1438b7a5 100644
--- a/rpc/src/v1/tests/mocked/signing_unsafe.rs
+++ b/rpc/src/v1/tests/mocked/signing_unsafe.rs
@@ -210,7 +210,7 @@ fn rpc_eth_send_transaction_with_bad_to() {
 		"id": 1
 	}"#;
 
-	let response = r#"{"jsonrpc":"2.0","error":{"code":-32602,"message":"Invalid params: prefix is missing."},"id":1}"#;
+	let response = r#"{"jsonrpc":"2.0","error":{"code":-32602,"message":"Invalid params: 0x prefix is missing."},"id":1}"#;
 
 	assert_eq!(tester.io.handle_request_sync(&request), Some(response.into()));
 }
```

**Comment by ordian** [comment]
> So, we are only running the tests on macOS?

No, github actions automatically cancels other job in case of a failure.

File: Cargo.lock [hunk 1]
```diff
@@ -969,28 +926,27 @@ version = "0.1.2"
...
  "ethereum-types",
- "home",
+ "home 0.3.4",
  "journaldb",
 ]
 
 [[package]]
 name = "docopt"
-version = "1.0.2"
+version = "1.1.0"
```
Comment (niklasad1) [inline Cargo.lock]: FYI, building this panic's for me on nightly , https://github.com/rust-lang/rust/issues/70302

File: Cargo.lock [hunk 2]
```diff
@@ -1105,9 +1061,12 @@ dependencies = [
 
 [[package]]
 name = "error-chain"
```
Comment (niklasad1) [inline Cargo.lock]: hrm, I thought we got rid of `error-chain` completely. Seems to related to `ethabi` though.

File: Cargo.lock (hunk 1 above)
Comment (vorot93) [reply Cargo.lock]: Fixed by https://github.com/rust-lang/rust/pull/70051

**Pull request merged**